CONTRIBUTING.md

```diff
@@ -4,6 +4,27 @@ Contributing to Skydive
 Your contributions are more than welcome. Please read the following notes to
 know about the process.
 
+Making Changes:
+---------------
+
+Before pushing your changes, please make sure the tests and the linters pass:
+
+* make fmt
+* make test
+* make test.functionals
```

Comment:
i feel it is better to mention that we need to export the SKYDIVE_ANALYZERS before running the tests? if not require ignore this :)